player.py

```diff
@@ -119,3 +120,35 @@ def follow_mouse(self, mouse_x: float, mouse_y: float):
         else:
             new_angle = new_angle - SPRITE_PLAYER_INIT_ANGLE
         self.angle = new_angle
+
+    def update(self, move_dir):
```

Comment:
Zero divison breaking the game and probably limit the rotation @MrTanoshii

Reply:
Fixed in 5cc198f71dd6bd96e94dab8f23e72ba5fd40089a